test/release/examples/README.testing

```diff
@@ -66,6 +66,7 @@ For a list of other test system options, use:
 For more information
 --------------------
 
-For complete documentation on the testing system, refer to the
-comments at the top of the start_test script.
+More information is provided in the TestSystem document, maintained with the
+Chapel source code and available on GitHub:
+https://github.com/chapel-lang/chapel/blob/master/doc/developer/bestPractices/TestSystem.rst
 
```

Comment:
The above is OK, although I'd prefer more brevity here, e.g.:

--------------------
For more information
--------------------

See:
  https://github.com/chapel-lang/chapel/blob/master/doc/developer/bestPractices/TestSystem.rst